frontend/src/scenes/trends/Trends.tsx

```diff
@@ -72,21 +72,23 @@ export function TrendInsight({ view }: Props): JSX.Element {
                     {renderViz()}
                 </div>
             )}
-            {breakdown && loadMoreBreakdownUrl && (
-                <div className="my-4 flex flex-col items-center">
-                    <div className="text-muted mb-2">
-                        For readability, <b>not all breakdown values are displayed</b>. Click below to load them.
+            {display !== ChartDisplayType.WorldMap && // the world map doesn't need this cta
```

Comment:
You are absolutely right that this showing up in the world map is a bug, but I'm worried be might be papering over a more severe issue. Because if phantom "Load more" appears for breakdowns by `$geoip_country_code`, are we sure that it's not also the case for all other breakdowns?

Reply:
Line chart trend with a breakdown on country code seems to do the right thing at least.

At first I don't get all countries and can load more. If I load more then some more appear, and I can load even more. If I load more a third time, the load more button disappears and the countries at the bottom of the list have a sum of 1, so I suspect that is the last.

Reply:
Hmm, perhaps the limit of breakdown values is lower than the actual number of different country codes? But this should not happen, `BREAKDOWN_VALUES_LIMIT_FOR_COUNTRIES` is set to 300, and there are definitely fewer than 300 [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Current_codes) codes in use. So maybe there's some integrity problem in `get_breakdown_prop_values()`?

Reply:
That is my worry

Reply:
I'm not not worried - but does the effort/reward on fixing this make sense given there's a big rewrite of trends happening anyway. @Gilbert09 can add this to the list of things to test afterwards :)